model/host/static_hosts.go

```diff
@@ -16,19 +16,23 @@ import (
 // This is called in the scheduler, and marks any static host in the system that
 // was removed from the distro as "terminated".
 //
-// Previously this oepration marked these hosts as "decommissioned,"
+// Previously this operation marked these hosts as "decommissioned,"
 // which is not a state that makes sense for static hosts.
 //
-// If the distro is the empty string ("") then this operation affects
-// all distros.
-func MarkInactiveStaticHosts(activeStaticHosts []string, distroID string) error {
+// If the distro is the empty string ("") then this operation affects all distros.
+// If distro aliases are included, then this operation affects also hosts with the alias.
+func MarkInactiveStaticHosts(activeStaticHosts []string, distroID string, distroAliases []string) error {
```

Comment:
The parameters would be cleaner if it took an entire distro rather than 2 separate fields. All the callers can be easily modified to pass the distro

Reply:
Done